code/player/TTTPlayer.RPCs.cs

```diff
@@ -13,6 +13,18 @@ private void ClientShowFlashlightLocal(bool shouldShow)
             ShowFlashlight(shouldShow);
         }
 
+        [ClientRpc]
+        public static void ClientOnPlayerCarriableItemPickup(TTTPlayer player, Entity carriable)
+        {
+            Event.Run("tttreborn.player.carriableitem.pickup", carriable as ICarriableItem);
+        }
+
+        [ClientRpc]
+        public static void ClientOnPlayerCarriableItemDrop(TTTPlayer player, Entity carriable)
+        {
+            Event.Run("tttreborn.player.carriableitem.drop", carriable as ICarriableItem);
+        }
```

Comment:
Does this have to be static if `player` is never used?

Reply:
good catch.